types/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// Minimum TypeScript Version: 3.2
+
+import {Node} from 'unist'
+import {TOCOptions} from 'mdast-util-toc'
+
+declare namespace remarkToc {
+  type TOCSettings = Pick<TOCOptions, 'heading' | 'maxDepth' | 'tight' | 'skip'>;
+}
+
+declare function remarkToc(
```

Comment:
Could you add TSDoc comments? https://github.com/syntax-tree/hast-util-to-html/pull/17/files#diff-88bec6beae84369c0376b56b3bb88fe1R171-R176 should be a good example.

Reply:
I added some TSDoc comments, but wasn't sure what else to add, other than what is in the readme 😅